astro/src/content/docs/get-started/core-concepts/collected-metrics.mdx

```diff
@@ -0,0 +1,535 @@
+    "webhookJwtRefreshTokenRevoke": 4,
+    "webhookJwtRefreshTokenRevokeTransactional": 6,
+    "webhookJwtRefreshTransactional": 6,
+    "webhookKickstartSuccess": 1,
+    "webhookUserAction": 4,
+    "webhookUserBulkCreate": 4,
+    "webhookUserBulkCreateTransactional": 6,
+    "webhookUserCreate": 4,
+    "webhookUserCreateComplete": 4,
+    "webhookUserCreateTransactional": 6,
+    "webhookUserDeactivate": 4,
+    "webhookUserDeactivateTransactional": 6,
+    "webhookUserDelete": 4,
+    "webhookUserDeleteComplete": 5,
+    "webhookUserDeleteTransactional": 6,
+    "webhookUserEmailUpdate": 4,
+    "webhookUserEmailVerified": 4,
+    "webhookUserEmailVerifiedTransactional": 6,
+    "webhookUserIdentityProviderLink": 6,
+    "webhookUserIdentityProviderUnlink": 4,
+    "webhookUserLoginFailed": 4,
+    "webhookUserLoginFailedTransactional": 6,
+    "webhookUserLoginIdDuplicateCreate": 4,
+    "webhookUserLoginIdDuplicateUpdate": 5,
+    "webhookUserLoginNewDevice": 4,
+    "webhookUserLoginNewDeviceTransactional": 6,
+    "webhookUserLoginSuccess": 5,
+    "webhookUserLoginSuccessTransactional": 5,
+    "webhookUserLoginSuspicious": 5,
+    "webhookUserLoginSuspiciousTransactional": 5,
+    "webhookUserPasswordBreach": 4,
+    "webhookUserPasswordBreachTransactional": 6,
+    "webhookUserPasswordResetSend": 4,
+    "webhookUserPasswordResetStart": 4,
+    "webhookUserPasswordResetSuccess": 4,
+    "webhookUserPasswordUpdate": 4,
+    "webhookUserReactivate": 5,
+    "webhookUserReactivateTransactional": 6,
+    "webhookUserRegistrationCreate": 4,
+    "webhookUserRegistrationCreateComplete": 4,
+    "webhookUserRegistrationCreateTransactional": 6,
+    "webhookUserRegistrationDelete": 4,
+    "webhookUserRegistrationDeleteComplete": 5,
+    "webhookUserRegistrationDeleteTransactional": 6,
+    "webhookUserRegistrationUpdate": 4,
+    "webhookUserRegistrationUpdateComplete": 4,
+    "webhookUserRegistrationUpdateTransactional": 6,
+    "webhookUserRegistrationVerified": 4,
+    "webhookUserRegistrationVerifiedTransactional": 6,
+    "webhookUserTwoFactorMethodAdd": 3,
+    "webhookUserTwoFactorMethodRemove": 5,
+    "webhookUserUpdate": 4,
+    "webhookUserUpdateComplete": 4,
+    "webhookUserUpdateTransactional": 6
+  }
+}
+```
+</details>
+
+You can see here that we are collecting information about feature usage.
+
+We will never collect anything personally identifiable. We may collect new metrics as we add new features, but those will be outlined in our release notes.
+
+Here's the breakdown of who we will collect data from, and how it will happen:
+
+-   Self-hosted: You get to choose whether or not you want to share your data with us. You can turn usage data collection off by API or in the admin UI. You can also block the outbound request at the network--just block the hostname https://usage-stats.fusionauth.io. If you decide to share usage data, know that we'll handle your data with the highest level of care and security.
```

Comment:
```suggestion
-   Self-hosted: You get to choose whether or not you want to share your data with us. You can turn usage data collection off by API, setup wizard, or in the admin UI, under XXX. You can also block the outbound request at the network--just block the hostname https://usage-stats.fusionauth.io. If you decide to share usage data, we will protect it.
```

where you need to replace XXX with the path to the admin UI screen where they can turn it off.

Also suggested change about the protection to make it a bit less flowery :) .